ecosystem/python/sdk/aptos_sdk/account_address.py

```diff
@@ -25,6 +25,8 @@ class AccountAddress:
     def __init__(self, address: bytes):
         self.address = address
 
+        # Note: The correctness of is_special relies on this check. Be careful when
+        # changing anything about this function.
```

Comment:
I don't understand the usefulness of this comment. If it isn't 32-bytes it isn't a valid bytes-based address.

Reply:
We went from 16 to 32 at one point, changes happen, I'd rather be defensive. I don't mind too much though, I can remove it if you want.